Makefile

```diff
@@ -68,70 +92,92 @@ vet:
-	$(CONTAINER_CMD) push ${API_IMAGE}
+	$(CONTAINER_CMD) tag bazel/cmd/forklift-api:forklift-api-image $(API_IMAGE)
+	$(CONTAINER_CMD) push $(API_IMAGE)
 
 build-validation-image:
 	bazel run validation:forklift-validation-image
 
 push-validation-image: build-validation-image
-	$(CONTAINER_CMD) tag validation:forklift-validation-image ${VALIDATION_IMAGE}
-	$(CONTAINER_CMD) push ${VALIDATION_IMAGE}
+	$(CONTAINER_CMD) tag bazel/validation:forklift-validation-image $(VALIDATION_IMAGE)
+	$(CONTAINER_CMD) push $(VALIDATION_IMAGE)
 
 build-operator-image:
 	bazel run operator:forklift-operator-image
 
 push-operator-image: build-operator-image
-	$(CONTAINER_CMD) tag operator:forklift-operator-image ${OPERATOR_IMAGE}
-	$(CONTAINER_CMD) push ${OPERATOR_IMAGE}
+	$(CONTAINER_CMD) tag bazel/operator:forklift-operator-image $(OPERATOR_IMAGE)
+	$(CONTAINER_CMD) push $(OPERATOR_IMAGE)
 
 build-virt-v2v-image:
 	bazel run virt-v2v:forklift-virt-v2v
 
 push-virt-v2v-image: build-virt-v2v-image
-	$(CONTAINER_CMD) tag virt-v2v:forklift-virt-v2v ${VIRT_V2V_IMAGE}
-	$(CONTAINER_CMD) push ${VIRT_V2V_IMAGE}
+	$(CONTAINER_CMD) tag bazel/virt-v2v:virt-v2v-image $(VIRT_V2V_IMAGE)
+	$(CONTAINER_CMD) push $(VIRT_V2V_IMAGE)
 
 build-operator-bundle-image:
-	bazel run operator:forklift-operator-bundle-image --action_env CONTROLLER_IMAGE=${CONTROLLER_IMAGE} --action_env VALIDATION_IMAGE=${VALIDATION_IMAGE} --action_env OPERATOR_IMAGE=${OPERATOR_IMAGE} --action_env VIRT_V2V_IMAGE=${VIRT_V2V_IMAGE} --action_env API_IMAGE=${API_IMAGE}
+	bazel run operator:forklift-operator-bundle-image \
+		--action_env VERSION=$(VERSION) \
+		--action_env NAMESPACE=$(NAMESPACE) \
+		--action_env CHANNELS=$(CHANNELS) \
+		--action_env DEFAULT_CHANNEL=$(DEFAULT_CHANNEL) \
+		--action_env OPERATOR_IMAGE=$(OPERATOR_IMAGE) \
+		--action_env MUST_GATHER_IMAGE=$(MUST_GATHER_IMAGE) \
+		--action_env MUST_GATHER_API_IMAGE=$(MUST_GATHER_API_IMAGE) \
+		--action_env UI_IMAGE=$(UI_IMAGE) \
+		--action_env UI_PLUGIN_IMAGE=$(UI_PLUGIN_IMAGE) \
+		--action_env VALIDATION_IMAGE=$(VALIDATION_IMAGE) \
+		--action_env VIRT_V2V_IMAGE=$(VIRT_V2V_IMAGE) \
+		--action_env CONTROLLER_IMAGE=$(CONTROLLER_IMAGE) \
+		--action_env API_IMAGE=$(API_IMAGE) \
+		--action_env VIRT_V2V_DONT_REQUEST_KVM=$(VIRT_V2V_DONT_REQUEST_KVM)
 
 push-operator-bundle-image: build-operator-bundle-image
-	 $(CONTAINER_CMD) tag operator:forklift-operator-bundle-image ${OPERATOR_BUNDLE_IMAGE}
-	 $(CONTAINER_CMD) push ${OPERATOR_BUNDLE_IMAGE}
+	 $(CONTAINER_CMD) tag bazel/operator:forklift-operator-bundle-image $(OPERATOR_BUNDLE_IMAGE)
+	 $(CONTAINER_CMD) push $(OPERATOR_BUNDLE_IMAGE)
 
 build-operator-index-image:
-	bazel run operator:forklift-operator-index-image --action_env REGISTRY=${REGISTRY} --action_env REGISTRY_ACCOUNT=${REGISTRY_ACCOUNT} --action_env REGISTRY_TAG=${REGISTRY_TAG}
+	bazel run operator:forklift-operator-index-image \
+		--action_env VERSION=$(VERSION) \
+		--action_env NAMESPACE=$(NAMESPACE) \
```

Comment:
the namespace is not used by the operator-index, no?

Reply:
True, I will fix it in the next push.